hack/lint_all_zarf_packages.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+ZARF_BIN=$1
+LINT_SRC_TEST=$2
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd $SCRIPTPATH
```

Comment:
suggestions from `shellcheck`

```suggestion
cd "$SCRIPTPATH" || exit
```